### ISSUE TITLE
Accuracy too low, any sugguestions?

### ISSUE THREAD

**Issue by LateTensor** [Canjie-Luo/MORAN_v2#155]
Step by step guided by the README, i created an env almost the same as the proj.

the env is listed below:
conda 4.5.11
pytorch 1.12
cuda 11.7

After changing the path of datasets in the 'train.sh' 
Using the CVPR2016 and NIPS2014, i got the highest accuracy 79.43 after 9 Epochs iterations. Seems far away from the paper result.
Anyone has any ideas or suggestion? 3x a lot !

**Comment by LateTensor** [comment]
The python version is 3.10 preinstalled in the Ubuntu 22.04LTS. There are a few syntax difference with the project's code, and i have already changed to eleminate all warnings.
